From cf6e3076228f6e5a194b0da21ef29bad6000dc0d Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Tue, 7 May 2013 12:35:52 +0300
Subject: [PDK MR2 Temp] Issues with mmgr paramer reading

BZ: 117199

There is an issue with mmgr trying to read the config
parameters and crashing. We have not yet found the cause
of this glib function usage.

This temporray fix makes telephony available while the DEV
team looks in to this issue.

Change-Id: Ia61f1c32839f80c8a2ca7c7ca25d19f1d09e33bb
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 service/config.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/service/config.c b/service/config.c
index 1546523..1a96c0c 100644
--- a/service/config.c
+++ b/service/config.c
@@ -215,8 +215,8 @@ static e_mmgr_errors_t set_param(GKeyFile *fd, char *grp, set_param_t *param)
             param->set.copy(param->dest, src);
             ret = E_ERR_SUCCESS;
         } else {
-            LOG_ERROR("READ ERROR: (%s)", gerror->message);
-            g_error_free(gerror);
+            //LOG_ERROR("READ ERROR: (%s)", gerror->message);
+            //g_error_free(gerror);
         }
     }
     param->set.display(param);
-- 
1.7.4.1

